net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Client.xml

```diff
@@ -14,7 +14,7 @@
                     <Required>Y</Required>
                 </name>
                 <pubkey type="TextField">
-                    <Required>N</Required>
+                    <Required>Y</Required>
```

Comment:
needs version bump of model, see above

Reply:
Since I created the PR through GitHub via editing the file (and not a fork), I'm not quite sure how to add the model file to this PR.  Looks like it might be updated in one of the other PR.

Reply:
scroll up in the editor of this file, there is a `<version>...</version>` tag and increase number. :)